scripts/services/check-removed-services.js

```diff
@@ -0,0 +1,133 @@
+const fs = require('fs/promises');
+const path = require('path');
+const yaml = require('js-yaml');
+
+const YML_FILE_EXTENSION = '.yml';
+
+const servicesDir = path.resolve(__dirname, '../../services/');
+
+/**
+ * Converts a service name to lowercase and replaces special characters.
+ *
+ * @param {string} serviceName - The service name to be normalized.
+ * @returns {string} The normalized service name.
+ */
+const normalizeFileName = (serviceName) => serviceName.toLowerCase().replace(/[^a-z0-9.]/g, '');
+
+/**
+ * Reads and parses the services data from from a JSON file.
+ *
+ * @param {string} jsonFilePath - The path to the JSON file.
+ * @returns {Promise<object[]|null>} A promise that resolves to an array of services data objects from JSON file.
+ * if the JSON file is successfully read and parsed. Returns `null` if there's an error during the process.
+ */
+const getJsonObjects = async (jsonFilePath) => {
+    try {
+        const jsonFileContent = await fs.readFile(jsonFilePath);
+        const jsonObjects = JSON.parse(jsonFileContent);
+        return jsonObjects.blocked_services;
```

Comment:
if the JSON is valid but it doesn't have `blocked_services` prop, then `undefined` returned, so maybe it's better to use
```suggestion
        return jsonObjects.blocked_services || null;
```
since you specified `null` in your JSDoc

Reply:
returns `null` if there's an error during the process